sos/plugins/openstack_glance.py

```diff
@@ -44,6 +46,20 @@ def setup(self):
 
         self.add_copy_spec("/etc/glance/")
 
+        run_cmds = False
+        for os_var in ['OS_USERNAME', 'OS_PASSWORD', 'OS_TENANT_NAME']:
+            if os_var not in os.environ:
+                self.soslog.warning("%s not found in environment variables"
+                                    " which is required to run commands"
+                                    % (os_var))
+            else:
+                run_cmds = True
+
+        if run_cmds:
+            self.add_cmd_output(
+                "openstack image list --long",
+                suggest_filename="openstack_image_list--long")
```

Comment:
Don't set `suggest_filename` unless you are setting it to something other than the default mangled command name.